### ISSUE TITLE
Does backbone.paginator have a Pager or should I implement my own?

### ISSUE THREAD

**Issue by anoopmd** [backbone-paginator/backbone.paginator#299]
I am new to backbone.pagonator.

In Addy's book, there was a request pager.
Is this option still there after the two projects merged OR should I implement my own pager view depending on the pagination state?

Thanks in advance !

**Comment by wyuenho** [comment]
You don't need to implement anything. Just read the README in this package.

**Comment by anoopmd** [comment]
Sorry Jim. I could not find it.
I did go through the Readme. 

Let me explain. I am able to create a Pageable Collection and loaded it with data from server.
I was able to get the next and previous pages using the api. i.e collection.getNextPage() and collection.getNextPage().

But I am looking to implement the ui like the one below
<<First <Prev Next> Last>>

So my question is that whether I should render my own ui component and tie them to the collection or does Backbone.Paginator has an existing solution ??
